Source/Core/FrustumGeometry.js

```diff
@@ -0,0 +1,489 @@
+define([
+        './BoundingSphere',
+        './Cartesian3',
+        './Cartesian4',
+        './Check',
+        './ComponentDatatype',
+        './defaultValue',
+        './defined',
+        './Geometry',
+        './GeometryAttribute',
+        './GeometryAttributes',
+        './Matrix3',
+        './Matrix4',
+        './OrthographicFrustum',
+        './PerspectiveFrustum',
+        './PrimitiveType',
+        './Quaternion',
+        './VertexFormat'
+    ], function(
+        BoundingSphere,
+        Cartesian3,
+        Cartesian4,
+        Check,
+        ComponentDatatype,
+        defaultValue,
+        defined,
+        Geometry,
+        GeometryAttribute,
+        GeometryAttributes,
+        Matrix3,
+        Matrix4,
+        OrthographicFrustum,
+        PerspectiveFrustum,
+        PrimitiveType,
+        Quaternion,
+        VertexFormat) {
+    'use strict';
+
+    var PERSPECTIVE = 0;
+    var ORTHOGRAPHIC = 1;
+
+    /**
+     * Describes a frustum at the given position and orientation.
+     *
+     * @alias FrustumGeometry
+     * @constructor
+     *
+     * @param {Object} options Object with the following properties:
+     * @param {PerspectiveFrustum|OrthographicFrustum} options.frustum The frustum.
```

Comment:
Is it easy to compute the input to a box geometry given an orthographic frustum?  If so, perhaps this should only take a perspective frustum and be renamed.